graphistry/dep_manager.py

```diff
@@ -0,0 +1,27 @@
+import importlib
+
+DEPS = ['cu_cat']
+
+class DepManager:
+    def __init__(self):
+        self.pkgs = {}
+        self.deps()
+
+    def __getattr__(self, pkg):
```

Comment:
add return types?

Reply:
added `pkg:str` here to return type; i dont think this is related to the pytest `type check` errors below and in https://github.com/graphistry/pygraphistry/pull/489.

but i also dont know why this `arrow_uploader` error is coming up

Reply:
any idea about this type check error, seemingly involving arrow_uploader? another PR pings this same error, not sure what to make of it

Reply:
Already fixed in main